web/client/translations/data.en-US.json

```diff
@@ -2987,41 +2989,42 @@
             "favouritesTitle": "Favourites",
             "allTitle": "All",
             "filterPlaceholder": "Filter map templates...",
-            "transfer": "Replace map with this template",
-            "merge": "Add this template to map",
+            "transfer": "Replace map with this map template",
+            "merge": "Add this map template to the map",
             "favouriteAdd": "Add to favourites",
             "favouriteRemove": "Remove from favourites",
             "emptyTitle": "No Result",
-            "emptyDescription": "Entered filter does not match name or description of any of templates",
-            "noTemplatesTitle": "No templates found",
-            "noTemplatesDescription": "Current context contains no templates",
+            "emptyDescription": "Entered filter does not match name or description of any of map template",
+            "noTemplatesTitle": "No map templates found",
+            "noTemplatesDescription": "Current context contains no map templates",
             "confirmReplaceTitle": "Replace map content",
-            "confirmReplaceMessage": "You will replace all layers and map configuration with the selected template by clicking on replace button",
+            "confirmReplaceMessage": "You will replace all layers and map configuration with the selected map template by clicking on the 'Replace' button.",
             "confirmReplaceConfirmButton": "Replace"
         },
         "mapCatalog": {
-            "title": "Map Catalog",
-            "filterPlaceholder": "Search maps...",
-            "deleteConfirmContent": "Are you sure you want to delete \"{mapName}\" map?",
+            "title": "Applications Catalog",
+            "tooltip": "Load an application with specific functionnalities and map templates.",
+            "filterPlaceholder": "Search applications...",
+            "deleteConfirmContent": "Are you sure you want to delete \"{mapName}\" application?",
             "tooltips": {
                 "edit": "Edit properties",
                 "delete": "Delete resource",
                 "share": "Share"
             },
             "deletedMap": {
-                "title": "Deleted Map",
-                "message": "The map was successfully deleted"
+                "title": "Deleted application",
+                "message": "The application was successfully deleted"
             },
             "updatedMap": {
-                "title": "Saved Map",
-                "message": "The map was successfully saved"
+                "title": "Saved application",
+                "message": "The application was successfully saved"
             },
-            "deleteError": "Unexpected error occurred while deleting the map",
-            "updateError": "Unexpected error occurred while saving the map"
+            "deleteError": "Unexpected error occurred while deleting the application.",
+            "updateError": "Unexpected error occurred while saving the application."
```

Comment:
For mapstore these are simply maps. If you want to use different semantics for your application, I suggest to customize these in geOrchestra translation files. 
What do you thing @tdipisa ?

Reply:
I agree @offtherailz, these should be provided as custom translations.

Reply:
Hello.

Sorry @tdipisa I wanted to send you an email yesterday to warn you about our aim to rename "map catalog" into "application". But as @offtherailz told previously : this concept is only existing now in MapStore georchestra.

So :

- I make new commits in this PR to keep "map"
- I open a new branch + PR on MS-geOrchestra to customize only this changement (but how it will interact when you'll update this file ??)
- remove `.` in the EN tooltip phrases
- I'll do my best for the others langages. no warranty...

Reply:
Sure !

Reply:
But it will be done next week...

Reply:
Points 1, 2 and 3 done. I'm on ES and DE.

Reply:
If it is suitable to you, please merge.